src/Conversion/ConversionCollection.php

```diff
@@ -135,9 +135,13 @@ public function getQueuedConversions(string $collectionName = ''): self
      */
     protected function addManipulationToConversion(Manipulations $manipulations, string $conversionName)
     {
-        $this->first(function (Conversion $conversion) use ($conversionName) {
+        if ($conversionName === 'all') {
```

Comment:
Compared to `all`, `*` has a lower change that it used as a collection name. 

Let's use `*` here.